python/vyos/kea.py

```diff
@@ -104,6 +104,14 @@ def kea_parse_options(config):
             }
         )
 
+    if 'capwap-access-controller' in config:
+        options.append(
+            {
+                'name': 'capwap-access-controller',
```

Comment:
```suggestion
                'name': 'capwap-ac-v4',
```

Using the built-in option name